index.js

```diff
@@ -1,211 +1,197 @@
-function openNav() {
-  document.getElementById("myNav").style.width = "100%";
-}
-  
-function closeNav() {
-  document.getElementById("myNav").style.width = "0%";
-}
-
-// Create an objects. A array of objects - My projects array
-
-var cardTitle = document.querySelectorAll(".card-body > h2");
-var cardImage = document.querySelectorAll(".snapshot");
-var cardInfo = document.querySelectorAll(".frame-2");
-var cardDesc = document.querySelectorAll(".discrip");
-var cardTech = document.querySelectorAll(".tags");
-var cardBtn = document.querySelectorAll(".cardButton");
-var cards = document.querySelectorAll(".card"); //comes from <arcticle class="card">
-
-console.log(cardTitle);
-console.log(cardImage);
-console.log(cardInfo);
-console.log(cardDesc);
-console.log(cardTech);
-console.log(cardBtn);
-console.log(cards);
-
-var cardObjects = [
-  card1 = {
+// Cards object creation
+const cardTitle = document.querySelectorAll('.card-body > h2');
+const cardImage = document.querySelectorAll('.section-img1');
+const cardInfo = document.querySelectorAll('.info');
+const cardDesc = document.querySelectorAll('.section-info');
+const cardTech = document.querySelectorAll('.program-btns');
+const cardBtn = document.querySelectorAll('.myBtn');
+let card1 = {};
+let card2 = {};
+let card3 = {};
+let card4 = {};
+const cardObjects = [
+  (card1 = {
```

Comment:
- [ ] Your project is approved, but fix this before moving on to the next milestone.
- Remove the text that shows up on the popup window underneath the images.
_See image below._

![image](https://user-images.githubusercontent.com/49621279/124360244-8fb21480-dbdd-11eb-81bf-0dc397b1ef92.png)

Reply:
Dear @Temmarie,

Thank you for your approval. I will fix your comments.

My regards,

Elmar